src/mocks.ts

```diff
@@ -39,8 +39,72 @@ type MockApiResponse = {
   status?: string;
   message?: string;
 };
+export const NAME_CURVE = 'P-256';
 
-const importRefreshKey = (refreshResponseKey: string) => {
+type CstgKeyPair = {
```

Comment:
Not sure about this name, as we're mixing a public and private key that don't belong to the same keypair.

Reply:
Yeah, I understand the name is not good here, but I still want to group these two into a parameter object. Hmmm, do you think name it with KeyBundleForDeriveSharedKey instead of CstgKeyPair is better??